### ISSUE TITLE
Create Game Scene

### ISSUE THREAD

**Issue by SilasReinagel** [EnigmaDragons/ProjectNeon#7]
Next Battle Button
Deck Builder Button

**Comment by SilasReinagel** [comment]
@0crat boost 1x

**Comment by SilasReinagel** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/EnigmaDragons/ProjectNeon/issues/7#issuecomment-529690834))

@SilasReinagel Job [#7](https://github.com/enigmadragons/projectneon/issues/7) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
> @0crat boost 1x ([here](https://github.com/EnigmaDragons/ProjectNeon/issues/7#issuecomment-529690790))

@SilasReinagel There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@1bed83ee-72bd-4b90-b626-859c1bf71176, thread: pool-5-thread-8
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-boosts.xsl says: The boost for job "gh:enigmadragons/projectneon#7" exists in boosts.xml, but the job is absent in the WBS among 3 jobs.
```

[0.49.8](https://github.com/zerocracy/farm): CID: [a35cf385-0959-496c-9b9e-5f2c2b46a63b](https://www.0crat.com/footprint/CN3V1M4AU/a35cf385-0959-496c-9b9e-5f2c2b46a63b), Type: "Set boost"

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by SilasReinagel** [comment]
@0crat boost 1x

**Comment by 0crat** [comment]
> @0crat boost 1x ([here](https://github.com/EnigmaDragons/ProjectNeon/issues/7#issuecomment-529696357))

@SilasReinagel Boost 1x was set for [#7](https://github.com/enigmadragons/projectneon/issues/7)

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
The job [#7](https://github.com/enigmadragons/projectneon/issues/7) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CN3V1M4AU/ebb0c52b-47a2-41bd-bd58-5275898e9f61); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
The architect of the project has changed; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) is not at this role anymore; @silasreinagel[/z](https://www.0crat.com/u/silasreinagel) is the architect now

**Comment by 0pdd** [comment]
@SilasReinagel 3 puzzles [#47](https://github.com/EnigmaDragons/ProjectNeon/issues/47), [#48](https://github.com/EnigmaDragons/ProjectNeon/issues/48), [#49](https://github.com/EnigmaDragons/ProjectNeon/issues/49) are still not solved.

**Comment by 0crat** [comment]
Job was finished in 12 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
The job [#7](https://github.com/enigmadragons/projectneon/issues/7) is now out of scope

**Comment by 0pdd** [comment]
@SilasReinagel all 3 puzzles are solved here: [#47](https://github.com/EnigmaDragons/ProjectNeon/issues/47), [#48](https://github.com/EnigmaDragons/ProjectNeon/issues/48), [#49](https://github.com/EnigmaDragons/ProjectNeon/issues/49).
